packages/render-fidelity-tools/test/config.json

```diff
@@ -437,7 +439,8 @@
       "exclude": [
         "gltfpt",
         "babylon"
-      ]
+      ],
+      "renderSkybox": true
```

Comment:
Can you also add the skybox back to the furnace test and the directional light test? Their backgrounds are simple, but helpful.